llvm/lib/CodeGen/AsmPrinter/CodeViewDebug.cpp

```diff
@@ -893,6 +893,8 @@ static TypeIndex getStringIdTypeIdx(GlobalTypeTableBuilder &TypeTable,
   return TypeTable.writeLeafType(SIR);
 }
 
+// This just exists for backwards compatability for the deprecated MCTargetOptions::CommandLineArgs
+// It assumed a clang compiler frontend
 static std::string flattenCommandLine(ArrayRef<std::string> Args,
```

Comment:
Remove this function. We'll let clients flatten their arguments as they wish. Please ensure that would work with rustc.